crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -782,6 +783,8 @@ pub struct SuiTransactionBlockEffectsV1 {
     /// The set of transaction digests this transaction depends on.
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
     pub dependencies: Vec<TransactionDigest>,
+    /// The error if the transaction failed with an abort.
+    pub abort_error: Option<Abort>,
```

Comment:
I'd probably slap a `#[serde(default, skip_serializing_if = "Option::is_none")]` on this